silkrpc/commands/debug_api_test.cpp

```diff
@@ -60,7 +60,7 @@ class DummyCursor : public silkrpc::ethdb::CursorDupSort {
         co_return;
     }
 
-    asio::awaitable<KeyValue> seek(const silkworm::ByteView& key) override {
+    asio::awaitable<KeyValue> seek(silkworm::ByteView key) override {
```

Comment:
why does passing by reference cause unit tests to fail?

Reply:
because pass-by-ref leads to undefined behavior if the object passed by-ref goes out of scope, so it's a powerful optimization technique in some cases (large objects) but requires extreme caution, see the other conversations for additional information

BTW, `silkworm::ByteView` is a template specialization of `std::basic_string_view` which should be passed by-value idiomatically, see [here](https://isocpp.org/blog/2021/11/three-reasons-to-pass-stdstring-view-by-value-arthur-odwyer?s=09)